modules/client/variables.tf

```diff
@@ -79,26 +79,31 @@ variable "server_address" {
 
 variable "client_ssh_allow_ranges" {
   description = "List of CIDRs that will be allowed ssh access to forseti server"
-  type        = "list"
+  type        = list(string)
   default     = ["0.0.0.0/0"]
 }
 
 variable "client_instance_metadata" {
   description = "Metadata key/value pairs to make available from within the client instance"
-  type        = "map"
+  type        = map(string)
   default     = {}
 }
 
 variable "client_tags" {
   description = "VM instance tags"
-  type        = "list"
+  type        = list(string)
   default     = []
 }
 
+
 variable "client_access_config" {
   description = "Client instance 'access_config' block"
-  type        = "map"
-  default     = {}
+  default     = []
+  type = list(object({
```

Comment:
Let's maintain the current interface of a single object rather than a list.

Reply:
Oops, I'll fix it now